### DISCUSSION TITLE
Determine the environment from config file.

### DISCUSSION THREAD

**Discussion by ankuagarwal** [dropwizard/dropwizard#4755] [General]
Hi,
I have 2 configuration (staging and production) yml files in my Dropwizard project. I need to determine which environment (staging or production) I am running the application. I was thinking if I could get the filename somehow, I should be able to set the environment in global variable.

Any idea how could I achieve this.

Thanks

**Comment by joschi** [comment]
How about adding a field to your configuration class which specifies the environment (could be a string or an enum) and read it in your application?

**Comment by ankuagarwal** [comment]
That's one way of doing it. But then will have to maintain this config for all environment.

**Comment by joschi** [comment]
@anandagarwaal-zz That's exactly what you described, isn't it?

> I have 2 configuration (staging and production) yml files in my Dropwizard project.

**Comment by ankuagarwal** [comment]
@joschi I'll try your approach by creating field which determine which type of config it is.
